app/views/accessibilityStatement.scala.html

```diff
@@ -0,0 +1,58 @@
+@()(implicit messages: Messages)
+
+@main(Messages("accessibility.header")) {
+@defining(play.core.PlayVersion.current) { version =>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">@Messages("accessibility.title")</h1>
```

Comment:
Wow, the messages file makes it hard to work with text-heavy pages like this! I'm going to raise this with the whole dev team, because I was never sure if the messages file was all that useful when we're fairly sure TDR will never be multilingual.

No need to do anything differently for this PR, though. If we remove the messages then it will happen later.

Reply:
I know right! 😅  I didn't think it would be so difficult to come up with various naming conventions for all the text, but we'll probably need to think of a better solution in the long run.